decidim-budgets/app/types/decidim/budgets/project_type.rb

```diff
@@ -16,7 +16,7 @@ module Budgets
       field :id, !types.ID, "The internal ID for this project"
       field :title, Decidim::Core::TranslatedFieldType, "The title for this project"
       field :description, Decidim::Core::TranslatedFieldType, "The description for this project"
-      field :budget, types.Int, "The budget for this project"
+      field :budget_amount, types.Int, "The budget for this project"
```

Comment:
I think that the description "The budget for this project" could be used for a field with a reference to the budget object and for this field. Maybe "The budget amount for this project" would avoid this?

Reply:
Yes, agree 👍